examples/motor_pca9685_stepper_motor.py

```diff
@@ -23,10 +23,11 @@
 # Motor 3 is channels 3 and 4 with 2 held high.
 # Motor 4 is channels 5 and 6 with 7 held high.
 
-pca.channels[7].duty_cycle = 0xffff
-pca.channels[2].duty_cycle = 0xffff
-stepper_motor = stepper.StepperMotor(pca.channels[4], pca.channels[3], # Motor 3
-                                     pca.channels[5], pca.channels[6]) # Motor 4
+pca.channels[7].duty_cycle = 0xFFFF
+pca.channels[2].duty_cycle = 0xFFFF
+stepper_motor = stepper.StepperMotor(
+    pca.channels[4], pca.channels[3], pca.channels[5], pca.channels[6]  # Motor 3
```

Comment:
Please remove the comments here as they are explained above and no longer make sense with the formatting change.